ansible-fmw/roles/dojo-wls-lifecycle/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
plz kindly update `README` , this content is autogenerated ;)

Reply:
This was a blank role, so we have deleted it and pushed again.